### PR TITLE
Deobfuscate packet flag comment a bit

### PR DESCRIPTION
**Pull request by ChillerDragon**

### CODE REVIEW

**Review by heinrich5991 — APPROVED** [review]
Thanks.

**Comment by ChillerDragon** [comment]
Oh I force pushed too late. I added the "a" back but not sure if that is really needed.

https://github.com/heinrich5991/libtw2/blob/480b603dbf36c9ad5bb2f281aeefbf0502b0fe87/doc/packet.md

> NOTE: padding must be zeroed, it's incorrectly used as part of the ack field while unpacking in the reference implementation.

File: src/engine/shared/network.h [hunk 1]
```diff
@@ -19,9 +19,19 @@ class CPacker;
...
-			0.6.5: ORNCTUAA
+			0.6.5: ORNCT-AA
 			0.7:   --NORCAA
 
+		O = flag compression
+		R = flag resend
+		N = flag connless
+		C = flag control
+		T = flag token (0.6.5 only not supported by ddnet)
+		- = unused
```
Comment (heinrich5991) [inline src/engine/shared/network.h:30]:
```suggestion
		- = unused, should be zero
```

**Comment by heinrich5991** [comment]
> You want to kick the "a" and only use "-" should be zero?

Nah, I like your version.

**Pull request merged**